index.html

```diff
@@ -155,9 +155,11 @@
 enables verifiable, decentralized digital identity. A <a>DID</a> identifies
 any subject (e.g., a person, organization, thing, data model, abstract entity, etc.)
 that the controller of the <a>DID</a> decides that it 
-identifies. These new identifiers are designed to enable the controller 
-of a <a>DID</a> to prove control over it and to be implemented independently 
-of any centralized registry, identity provider, or certificate authority. 
+identifies.
+DIDs are implemented
+independently of any centralized registry, identity provider, or
+certificate authority but none the less enable the controller of
```

Comment:
How about, "but nonetheless are designed to enable the controller..."

Reply:
Better, but the beginning of the sentence still needs work. You're going to get push back on:

"DIDs are implemented independently" ... as opposed to something that indicates that the design *enables* them to be implemented that way.